gusto/initialisation_tools.py

```diff
@@ -0,0 +1,94 @@
+"""
+A module containing some tools for computing initial conditions, such
+as balanced initial conditions.
+"""
+
+from __future__ import absolute_import
+from firedrake import MixedFunctionSpace, TrialFunctions, TestFunctions, \
+    FacetNormal, inner, div, dx, ds_b, ds_t, DirichletBC, \
+    Expression, Function, Constant, \
+    LinearVariationalProblem, LinearVariationalSolver
+
+
+def compressible_hydrostatic_balance(state, theta0, rho0,
+                                     top=True, pi_boundary=Constant(1.0),
```

Comment:
I'm not sure what the default is for the sphere, but using this for the slice I would want the default to be to set pi=1. on the bottom i.e. top=False and pi_boundary=Constant(1.0). Do these current defaults make sense on the sphere?